### ISSUE TITLE
Please add a way to disable the 'can one of the admins verify this patch' comment

### ISSUE THREAD

**Issue by jschmid1** [jenkinsci/ghprb-plugin#608]
If you have rather strict rules on whether you are allowed to trigger tests or not, you can't enable the `Build every pull request automatically without asking (Dangerous!).` option which would disable the warning.

I found the option to change the default message, which is nice but simply not enough. We have multiple jobs with the jenkins pr builder that point to one repo and can be triggered by different trigger phrases. By not having the possiblity to disable these comments every new contributor gets flooded with these messages.

So my request would be to have a **job**-wide(not system-wide) switch to toggle these warnings.

**Comment by manovotn** [comment]
Seconding this^.
We have the exact same "problem" - having three jobs polling the PRs, this already floods the PR comments.

**Comment by slehan** [comment]
Yeah, in our case (and I think many others) this comment ends up being spammy and not necessarily required.

**Comment by bjoernhaeuser** [comment]
Can you attach a screenshot of your settings please? Or even better the config file you are using? Both system and job configuration.

**Comment by pkubatrh** [comment]
Yes, this would really be nice to have

**Comment by EdwardW2** [comment]
Has this been added yet? Want to get Jenkins to not comment on any PRs at all and only leave its status there and thats it.

**Comment by ArthiIreddy** [comment]
Any update on this? although I whitelisted the users with the github user name the comment "Can one of the admins verify this patch?" is still auto generated?

**Comment by dooleyb1** [comment]
Would also love if this feature could be added.

**Comment by wimglenn** [comment]
Can one of the admins verify this patch?  🙄

**Comment by frenzymadness** [comment]
It would be very nice to have!

**Comment by Pymann** [comment]
I would also like to disable this message. It seems, that there are also differences within versions. In an older one I was able to disbale it by enabling "build every pull-request (dangeorus)" and a whitelist of users.

**Comment by peternied** [comment]
We've been running into this issue with our Jenkins setup, such as on this PR https://github.com/opensearch-project/OpenSearch/pull/1943
